### ISSUE TITLE
[DataCap Application] <MetoldDigital> - <Enterprise data>

### ISSUE THREAD

**Issue by MetoldDigital** [OpenGate01/Allocator-Pathway-OpenGate01#14]
### Version

1

### DataCap Applicant

MetoldDigital

### Project ID

04

### Data Owner Name

MetoldDigital

### Data Owner Country/Region

Australia

### Data Owner Industry

Life Science / Healthcare

### Website

http://www.metold.digital/

### Social Media Handle

https://x.com/MeToldX

### Social Media Type

Twitter

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

600TiB

### Number of replicas to store

8

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

None

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
MeTold Tech is an innovative technology company at the forefront of blending profound human emotional needs with cutting-edge blockchain technology. We are dedicated to creating a secure, private, and emotionally rich online memorial space for commemorating and remembering departed loved ones. MeTold Tech is not just a provider of technology; it’s a platform for emotional resonance, bringing memories and stories to eternal life in the digital realm.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Commemorative videos and photos for users
Content of stories and letters written or recorded by users
Visual images of digital gifts designed by users
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

None

### Please share a sample of the data

```text
None
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### If you chose not to confirm, what was the reason

```text
These datas are all belong to users,it's private
```

### What is the expected retrieval frequency for this data

Weekly

### For how long do you plan to keep this dataset stored on Filecoin

Permanently

### In which geographies do you plan on making storage deals

Greater China, North America, Europe, Australia (continent)

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS

### How did you find your storage providers

Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
will disclose in the comment when sps are confirmed
```

### How do you plan to make deals to your storage providers

Boost client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by OpenGate01** [comment]
Thank you for your recognition of Filecoin. We would like to schedule a time for an on-site visit, if possible.

**Comment by MetoldDigital** [comment]
Thank you for your reply,if you want to have an on-site visit,here is our address :Level 5, 123 George Street, Sydney, NSW 2121.
Or you could contact with us on line at first,we will be in the office from 10:00 to 17:00 on weekdays, so you can confirm a time to meet with us.

**Comment by OpenGate01** [comment]
We look forward to visiting in person. I believe that storing elderly people's data on Filecoin is actually very meaningful for every family. This storage method is both private and secure.

**Comment by MetoldDigital** [comment]
> We look forward to visiting in person. I believe that storing elderly people's data on Filecoin is actually very meaningful for every family. This storage method is both private and secure.

Thank you for your anticipation
 we are also looking forward to meeting you. Indeed, storing elderly people's data on Filecoin is very important for every family.
Can we have an offline meeting this Wednesday afternoon? You are always welcome to come :)

**Comment by OpenGate01** [comment]
Of course, we've heard that the Australian government provides each elderly person with a palliative care fund of AUD 35,000, and the future prospects of your company are very promising. Filecoin storage is a very secure method, which also provides excellent protection for the privacy of the elderly.

**Comment by MetoldDigital** [comment]
> Of course, we've heard that the Australian government provides each elderly person with a palliative care fund of AUD 35,000, and the future prospects of your company are very promising. Filecoin storage is a very secure method, which also provides excellent protection for the privacy of the elderly.

Yes, Australia places great emphasis on care for the elderly, and we are also providing services related to digital generation. We plan to generate digital avatars from these videos. If AI needs to access your data, how can it do so?

**Comment by OpenGate01** [comment]
We recommend that keeping 6-8 backups of your data is sufficient.

**Comment by MetoldDigital** [comment]
Okay, thank you for your suggestions! Is there anything else you need us to handle?

**Comment by OpenGate01** [comment]
Since you have already had initial contact with us and we can confirm that the submission is made under your own volition, as the person in charge, I still need to understand some additional information.
Are you familiar with the data preparation process? Could you briefly explain how you prepare the data?
Do you have a clear plan for your cooperating SP nodes?

**Comment by MetoldDigital** [comment]
> We recommend that keeping 6-8 backups of your data is sufficient.

 Thank you. Regarding the data preparation process, we currently store most of our data on private servers. We prepare the data based on its type. For example, for large data files, we prefer to back up and store them through physical transfer, while smaller files can be transferred via the network.
As for our cooperating SPs, we currently have f03179555 SG, f03179570 SG, f02830321 Ho Chi Minh City, and f02814393 Hanover. We mainly cooperate with high-quality SPs recommended by our current partners, and we plan to add more in the future.

**Comment by OpenGate01** [comment]
There was an error when attempting to approve. Please resubmit a form with the new address.

**Comment by MetoldDigital** [comment]
> There was an error when attempting to approve. Please resubmit a form with the new address.

Sorry for the inconvenience,we rusubmit an application here:
(https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/21)
Thanks!
